src/elements/content-sidebar/SidebarResponsiveness.scss

```diff
@@ -0,0 +1,154 @@
+@import '../common/variables';
+
+.u-md-bcs-SidebarNavSign-icon {
+    display: none;
+}
+
+.is-responsive-web {
+    @include breakpoint($medium-screen) {
+        .bcs-SidebarNavSign-icon {
+            display: none;
+        }
+
+        .u-md-bcs-SidebarNavSign-icon {
+            display: inline;
+            width: 20px;
+            height: 20px;
+            margin-top: 2px; //for visual alignment
+        }
+
+        .bcpr {
+            .bcpr-body {
+                position: static;
```

Comment:
Does this `static` position affect the way annotations are rendered on top of preview?

Reply:
Annotations are unaffected by the `position: static`:
![Screen Shot 2022-06-29 at 1 44 10 PM](https://user-images.githubusercontent.com/6400298/176541117-9762ae02-5668-416e-8f9f-751402068173.png)